Lombiq.VueJs/Views/_ViewImports.cshtml

```diff
@@ -1,4 +1,4 @@
-﻿@inherits OrchardCore.DisplayManagement.Razor.RazorPage<TModel>
+@inherits OrchardCore.DisplayManagement.Razor.RazorPage<TModel>
```

Comment:
I think these are some kind of a bug. So basically if I revert this I will get a red lined error for no reason. Because they are literally the same.
![image](https://github.com/Lombiq/Orchard-Vue.js/assets/92299130/dd71f2b9-eab2-49ef-b7c6-64e5100ddf96)